test/unit/abilities/user_permissions_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class Abilities::UserPermissionsTest < ActiveSupport::TestCase
+
+  def setup
+    @provider = Factory.create(:simple_provider)
```

Comment:
Since we are going to push anyway, we should change all the `Factory(...)` and `Factory.create(...)` in this file for `FactoryGirl.create(...)`